src/jackdaw/test/middleware.clj

```diff
@@ -15,9 +15,9 @@
   "Timing middleware. Adds timing information to the response map"
   [f]
   (fn [machine cmd]
-    (let [start (. System (nanoTime))
+    (let [start (System/nanotime)
           result (f machine cmd)
-          end (. System (nanoTime))
+          end (System/nanotime)
```

Comment:
There's a typo here. Should be `nanoTime`.

Reply:
thanks I'll fix it